tools/storage-usage-delta-calculator/src/main.rs

```diff
@@ -0,0 +1,39 @@
+use log::{debug, LevelFilter};
+use near_chain_configs::Genesis;
+use near_primitives::runtime::config::RuntimeConfig;
+use near_primitives::state_record::StateRecord;
+use node_runtime::Runtime;
+use std::fs::File;
+use std::io::Error;
+
+/// Calculates delta between actual storage usage and one saved in state
+/// output.json should contain dump of current state,
+/// run state-viewer --home ~/.near/mainnet/ dump_state
+/// to get it
+fn main() -> Result<(), Error> {
+    env_logger::Builder::new().filter(None, LevelFilter::Debug).init();
+
+    debug!("Start");
```

Comment:
Should we remove these?

Reply:
I amnpersonally concerned when I start some tool and it seemingly do nothing for 15 minutes

Reply:
For some reason it seems to be a single conversation and I cannot reply directly to comment above. Your approach may well be better if we only have one item in MigrationData, but how do you think we should deal with multiple such elements?